app/src/components/widgets/singleLevelWidgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx

```diff
@@ -83,6 +100,26 @@ export class PassingRatePerLaunch extends Component {
     this.chart.focus(id);
   };
 
+  onChartClick = (data) => {
+    const { widget, getStatisticsLink } = this.props;
+    const link = getStatisticsLink({
+      statuses: data.id === STATS_PASSED ? [PASSED] : [FAILED, INTERRUPTED, SKIPPED],
+      launchesLimit: widget.contentParameters.itemsCount,
+    });
+    const navigationParams = this.getDefaultParamsWidget(widget.appliedFilters[0].id);
+
+    this.props.navigate(Object.assign(link, navigationParams));
+  };
+
+  getDefaultParamsWidget = (filterId) => ({
```

Comment:
`getDefaultParamsWidget` sounds unclear. Please rename it to `getDefaultLinkParams` or `getCommonLinkParams`.